ts/features/bonusVacanze/screens/ActiveBonusScreen.tsx

```diff
@@ -450,21 +449,7 @@ const ActiveBonusScreen: React.FunctionComponent<Props> = (props: Props) => {
   );
 
   const renderFooterButtons = () =>
-    bonus && isBonusActive(bonus) ? (
-      renderBonusActiveButtons()
-    ) : (
-      <>
-        <BlockButtons
-          type="SingleButton"
-          leftButton={{
-            bordered: true,
-            title: I18n.t("global.buttons.cancel"),
-            onPress: props.goBack
-          }}
-        />
-        <View spacer={true} />
-      </>
-    );
+    bonus && isBonusActive(bonus) ? renderBonusActiveButtons() : undefined;
```

Comment:
```suggestion
    bonus && isBonusActive(bonus) && renderBonusActiveButtons();
```

Maybe would be cleaner? Or we may too remove `renderBonusActiveButtons` making `renderFooterButtons` rendering directly the footer component?